clash-lib/src/Clash/Backend/VHDL.hs

```diff
@@ -564,6 +567,12 @@ tyDec hwty = do
                  <+> parens (hsep (punctuate comma variantNames))
                  <> semi
 
+    MemBlob _ m ->
+      "type" <+> tyName hwty
+             <+> "is array (integer range <>) of std_logic_vector"
+             <> parens (int (m - 1) <+> "downto 0")
+             <> semi
+
```

Comment:
```suggestion
    MemBlob n m -> tyDec (Vector n (BitVector m))

```
Treat `MemBlob n m` as an alias for `Vector n (BitVector m)`. There's really no point to treat it any differently than an alias since the whole point of this exercise is to speed up compile time (and perhaps memory use in simulation)